kimchi/src/circuits/constraints.rs

```diff
@@ -52,14 +52,14 @@ pub struct FeatureFlags {
 /// The polynomials representing evaluated columns, in coefficient form.
 #[serde_as]
 #[derive(Clone, Serialize, Deserialize, Debug)]
-pub struct EvaluatedColumnCoefficients<F: PrimeField> {
+pub struct EvaluatedColumnCoefficients<const W: usize, F: PrimeField> {
```

Comment:
It might make these changes smaller if you set a default value for each of the generics, e.g. `const W: usize = COLUMNS`.

Reply:
This is a great idea